content/en/blog/_posts/2019-12-21-Kubernetes-on-MIPS.md

```diff
@@ -0,0 +1,143 @@
+| docker-ce on MIPS      | 18.09.8 | -                                                                                                                                                                                                                                                                                 |
+| metrics-server for CKE on MIPS | 0.3.2   | `registry.inspurcloud.cn/library/cke/kubernetes/metrics-server-mips64el:v0.3.2`                                                                                                                                                                                                   |
+| etcd for CKE on MIPS           | 3.2.26  | `registry.inspurcloud.cn/library/cke/etcd/etcd-mips64el:v3.2.26`                                                                                                                                                                                                                  |
+| pause for CKE on MIPS          | 3.1     | `registry.inspurcloud.cn/library/cke/kubernetes/pause-mips64el:3.1`                                                                                                                                                                                                               |
+| hyperkube for CKE on MIPS      | 1.14.3  | `registry.inspurcloud.cn/library/cke/kubernetes/hyperkube-mips64el:v1.14.3`                                                                                                                                                                                                       |
+| coredns for CKE on MIPS         | 1.6.5   | `registry.inspurcloud.cn/library/cke/kubernetes/coredns-mips64el:v1.6.5`                                                                                                                                                                                                          |
+| calico for CKE on MIPS         | 3.8.0   | `registry.inspurcloud.cn/library/cke/calico/cni-mips64el:v3.8.0` `registry.inspurcloud.cn/library/cke/calico/ctl-mips64el:v3.8.0` `registry.inspurcloud.cn/library/cke/calico/node-mips64el:v3.8.0` `registry.inspurcloud.cn/library/cke/calico/kube-controllers-mips64el:v3.8.0` |
+
+> Note: CKE is a kubernetes based cloud container engine launched by Inspur
+
+![K8S-MIPS Cluster Components](/images/blog/2019-12-21-Kubernetes-on-MIPS/k8s-mips-cluster-components.png)
+
+*Figure 2 K8S-MIPS Cluster Components*
+
+![CPU Architecture](/images/blog/2019-12-21-Kubernetes-on-MIPS/cpu-architecture.png)
+
+*Figure 3 CPU Architecture*
+
+![Cluster Node Information](/images/blog/2019-12-21-Kubernetes-on-MIPS/cluster-node-information.png)
```

Comment:
I'm wondering about nomenclature. What's shown here as a “slave node” is usually called either just a “node” or a “worker node”.

Not a vital thing, although the IT industry has been moving away ([example](https://tools.ietf.org/id/draft-knodel-terminology-00.html)) from using the word “slave” in this sense.

Reply:
Yes you're right, using 'slave' is a lack of rigour here. I'll update that later.BTW, I guess you mean this image [Figure 1](https://github.com/kubernetes/website/blob/77f2b226be8e6193391ef81a30e76ed8e192acf6/static/images/blog/2019-12-21-Kubernetes-on-MIPS/k8s-mips-cluster-components.png)

Reply:
Correct! +1 to @sftim

Reply:
Ah yes, Figure 1 was the one I meant.